RevenueCatUI/Data/PaywallViewConfiguration.swift

```diff
@@ -29,7 +29,7 @@ struct PaywallViewConfiguration {
         fonts: PaywallFontProvider = DefaultPaywallFontProvider(),
         displayCloseButton: Bool = false,
         introEligibility: TrialOrIntroEligibilityChecker? = nil,
-        purchaseHandler: PurchaseHandler
+        purchaseHandler: PurchaseHandler? = nil
```

Comment:
Instead of this being nullable can we initialize it to `default`? Also, why is it nullable now? We are not passing nil anywhere and this is internal

Reply:
This is reverted back into the state that it was originally 🤷‍♂️  It does kind of seem like it should be `.default` but I should play with it a bit more before making that change to figure out why it wasn't set as this